.github/pull_request_template.md

```diff
@@ -0,0 +1,13 @@
+## Links (Issue link or board ticket):
+<!--- Include links that could be useful to track this PR --->
+
+## What & Why:
+<!--- Describe the changes and why is useful --->
+<!--- Add screenshots if needed --->
+
+## Testing steps:
+<!--- What are the steps to make sure it is working? --->
+
+## Risks:
```

Comment:
my only concern with these sections is that it makes it harder for external people to start contributing. I mean, if someone wants to open a PR in a gem I maintain, it's more than welcome no matter what they write as the description

Reply:
Thanks for the feedback, I will update it with a simpler template. https://github.com/rails/rails/edit/master/.github/pull_request_template.md